stl/inc/xutility

```diff
@@ -2899,6 +2899,34 @@ namespace ranges {
 
     // VARIABLE ranges::prev
     inline constexpr _Prev_fn prev{_Not_quite_object::_Construct_tag{}};
+
+    // STRUCT ranges::equal_to
+    struct equal_to {
+        // clang-format off
+        template <class _Ty1, class _Ty2>
+            requires equality_comparable_with<_Ty1, _Ty2> // TRANSITION, GH-489
+        _NODISCARD constexpr bool operator()(_Ty1&& _Left, _Ty2&& _Right) const noexcept(noexcept(
+            static_cast<bool>(static_cast<_Ty1&&>(_Left) == static_cast<_Ty2&&>(_Right)))) /* strengthened */ {
+            return static_cast<bool>(static_cast<_Ty1&&>(_Left) == static_cast<_Ty2&&>(_Right));
+        }
+        // clang-format on
+
+        using is_transparent = int;
+    };
+
+    // STRUCT ranges::less
+    struct less {
+        // clang-format off
+        template <class _Ty1, class _Ty2>
+            requires totally_ordered_with<_Ty1, _Ty2> // TRANSITION, GH-489
+        _NODISCARD constexpr bool operator()(_Ty1&& _Left, _Ty2&& _Right) const noexcept(

```

Comment:
Nitpick: That line is differently formated than in all the other CPOs

Reply:
Good catch - I must have changed something here after applying the clang-format suppression.